src/main/java/net/sf/mzmine/modules/peaklistmethods/filtering/duplicatefilter/help/help.html

```diff
@@ -36,5 +51,11 @@ <h4>Method parameters</h4>
 
 </dl>
 
+
+        <p>
+        <h4>Aligned, gap-filled feature list prior and after duplicate row filter (new average).</h4>
+        <img src="prior.png" name="prior">

```

Comment:
The files you added are named prior.PNG and after.PNG - the capitalization does make a difference on unix platforms.